frontend/src/components/header.js

```diff
@@ -163,12 +163,12 @@ const Header = () => {
             </div>
           </div>
 
-          <div className="flex flex-row items-center gap-4 md:gap:6 [@media(min-width:1040px)]:gap-8 [@media(min-width:1440px)]:gap-11">
+          <div className="flex flex-row items-center justify-items-center gap-2 md:gap:4 [@media(min-width:1140px)]:gap-4 [@media(min-width:1280px)]:gap-6 [@media(min-width:1440px)]:gap-11">
             {browseList.map((item, index) => (
               <Link
                 key={`browseLink-${index}`}
                 to={item.route}
-                className={`${textStyle} ${
+                className={`${"min-w-max"} ${textStyle} ${
```

Comment:
```suggestion
                className={`min-w-max ${textStyle} ${
```

you can just  have a this.